model/proto/model.proto

```diff
@@ -137,8 +137,12 @@ message Span {
 }
 
 message TraceID {
-  uint64 low = 1;
-  uint64 high = 2;
+  uint64 low = 1 [
+    (gogoproto.jsontag) = "lo"
```

Comment:
?

Reply:
It was previously 

```
// TraceID is a random 128bit identifier for a trace
type TraceID struct {
	Low  uint64 `json:"lo"`
	High uint64 `json:"hi"`
}
```

So I figured doing this would fix the issue but it's not the case

Reply:
I'm taking a look. Just a sec.